desktop/transactionManager.js

```diff
@@ -88,6 +92,20 @@ class TransactionManager {
     return { transactions: this.transactions };
   };
 
+  updateTxContact = ({ newData, accountIndex }) => {
+    const { address, nickname } = newData;
+    const addrString = address.substring(2).toLowerCase();
+    this.transactions[accountIndex].data.forEach((tx, index) => {
```

Comment:
here you have to iterate through all transactions in app, not just txs of the current account